### ISSUE TITLE
How to match every app except for one?

### ISSUE THREAD

**Issue by kalirosenkreuz** [dunst-project/dunst#1384]
I would like every notification to play a sound, except for discord which has its own sound. How would I set the patterns to exclude discord?

**Comment by bynect** [comment]
There was an idea for negative matches in #1040 but it is not implemented. Thus you have to use some kind of workaround. for example you add a sound to a catchall notification rule and a different one to discord (after this one).

**Comment by stormytuna** [comment]
How do you actually do this? I have the following in my dunstrc but still get the sound notification when using the command `notify-send foo --app-name discord`

```
[play_sound]
appname="*"
script="/nix/store/1jrx7ybvb1n7iq31x9bgzyrzj1fyqpzm-dunst_playsound"

[play_sound_no_discord]
appname="discord"
script=""
```

**Comment by bynect** [comment]
> How do you actually do this? I have the following in my dunstrc but still get the sound notification when using the command `notify-send foo --app-name discord`
> 
> ```
> [play_sound]
> appname="*"
> script="/nix/store/1jrx7ybvb1n7iq31x9bgzyrzj1fyqpzm-dunst_playsound"
> 
> [play_sound_no_discord]
> appname="discord"
> script=""
> ```

you are right. I tried even with always_run_script turned off and this unfortunately does not work. We will have to implement negative matches.

**Comment by fedang** [comment]
#1536 allows for negative matches
